backend/de.metas.business/src/main/java/de/metas/inventory/impl/InventoryDAO.java

```diff
@@ -93,10 +105,55 @@ public void setInventoryLinesProcessed(@NonNull final InventoryId inventoryId, f
 				.execute();
 	}
 
+	@Override
+	public void setInventoryLinesCounted(@NonNull final InventoryId inventoryId, final boolean counted)
+	{
+		queryBL.createQueryBuilder(I_M_InventoryLine.class)
+				.addOnlyActiveRecordsFilter()
+				.addEqualsFilter(I_M_InventoryLine.COLUMNNAME_M_Inventory_ID, inventoryId)
+				.addNotEqualsFilter(I_M_InventoryLine.COLUMNNAME_IsCounted, counted)
+				.create()
+				.updateDirectly()
```

Comment:
what's the reason of updating directly?

are u aware that model interceptors won't be fired?

Reply:
Because there is no interceptors related and would be faster